modules/core/shared/src/main/scala/gem/math/MagnitudeValue.scala

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) 2016-2017 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package gem.math
+
+import cats.{ Order, Show }
+import cats.instances.int._
+
+/**
+ * Exact magnitude value represented as an int with the original value scaled up
+ * @param value This magnitude integral value
+ */
+final case class MagnitudeValue(val value: Int) extends Product with Serializable
```

Comment:
I see case clases defined both with/without `extends Product with Serializable`. Should we unify the criteria?